src/pybind/mgr/dashboard/services/cluster.py

```diff
@@ -33,3 +34,12 @@ def from_db(cls):
         If the status is not set, assume it is already fully functional.
         """
         return cls(status=mgr.get_store('cluster/status', cls.Status.POST_INSTALLED.name))
+
+    def get_capacity():
+        df = mgr.get('df')
+        df = dict(stats=partial_dict(
+            df['stats'],
+            ['total_avail_bytes', 'total_bytes',
+                'total_used_raw_bytes']
+        ))
```

Comment:
Let's create an explicit Model with a NamedTuple or [TypedDict](https://docs.python.org/3/library/typing.html#typing.TypedDict) class (this will require importing the typing-extensions package) and only return the data you really need explicitly:
```py
class ClusterCapacity(TypedDict):
  total_avail_bytes: int
  total_bytes: int
  total_used_raw_bytes: int

def get_capacity() -> ClusterCapacity:
  df = mgr.get('df')
  return ClusterCapacity(total_avail_bytes = df['total_avail_bytes']...
```
This may look more redundant, but given this is about an API (the REST API), it's key to have models that reflect the data types passed/returned.